lib/plugins/chat.js

```diff
@@ -161,7 +161,10 @@ function inject (bot, options) {
           messageCount: 0,
           acknowledged: Buffer.alloc(3),
           // 1.19.2 Chat Command packet also includes an array of last seen messages
-          previousMessages: []
+          previousMessages: bot._client._lastSeenMessages.map((e) => ({
```

Comment:
This whole if block from L 151-170 can be removed as command parsing is handled in node-minecraft-protocol's chat(). Seems mineflayer wasn't updated for the change, but this can just become 
```js
if (bot.supportFeature('signedChat') && message.startsWith('/') {
  bot._client.chat(message)
}
```

Reply:
Cool, that also fixed the issue I was having. I did however leave the return statement in since I assume that would cause it to send commands twice